meson.build

```diff
@@ -9,24 +9,30 @@ project('qrtr',
         ])
 
 prefix = get_option('prefix')
-systemd = dependency('systemd')
 
-systemd_system_unit_dir = get_option('systemd-unit-prefix')
-if systemd_system_unit_dir == ''
-        systemd_system_unit_dir = systemd.get_variable(
-                pkgconfig : 'systemdsystemunitdir',
+install_systemd_unit = get_option('systemd-service')
+if install_systemd_unit.auto() or install_systemd_unit.enabled()
+        systemd = dependency('systemd')
```

Comment:
Please use meson way of doing things. Add `required: false` and then use `if systemd.found()`